src/ModelContextProtocol/Shared/McpSession.cs

```diff
@@ -16,6 +18,13 @@ namespace ModelContextProtocol.Shared;
 /// </summary>
 internal sealed class McpSession : IDisposable
 {
+    private static readonly ActivitySource s_activitySource = new("ModelContextProtocol");
```

Comment:
Should this be tagged as Experimental?

Reply:
My vote is also no. While we will probably end up breaking people expecting a stable metrics API right now, we might not, And if we don't, why make them update their listeners? While not exactly the same, my arguments are similar to what's in the "Deprecating the "X-" Prefix and Similar Constructs in Application Protocols" RFC 

https://www.rfc-editor.org/rfc/rfc6648